src/PresentationalComponents/CreateImageWizard/WizardStepUploadAzure.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Form, FormGroup, Text, TextContent, TextInput, Title } from '@patternfly/react-core';
+import { CheckCircleIcon, ExternalLinkAltIcon, ResourcesEmptyIcon } from '@patternfly/react-icons';
+
+import './WizardStepUploadAzure.scss';
+
+const WizardStepUploadAzure = (props) => {
+    return (
+        <>
+            <TextContent className="textcontent-azure">
+                <Title headingLevel="h2">Upload to Azure</Title>
+                <Text>
+            Image Builder will send an image to an authorized Azure account.
+                </Text>
+                <Title headingLevel="h3">OAuth permissions</Title>
+                <Text>
+            In order to use Image Builder to push images to Azure, Image Builder must
+            be configured as an authorized application, and given the role of &quot;Contributor&quot;.<br />
+            Image Builder must be authorized by an account owner.<br />
+                    <a href="https://docs.microsoft.com/en-us/azure/active-directory/develop/v2-oauth2-auth-code-flow">
+                        <small>Learn more</small></a>
+                </Text>
+
+                <a href="https://login.microsoftonline.com/common/oauth2/authorize?client_id=b94bb246-b02c-4985-9c22-d44e66f657f4
+&redirect_uri=https%3A%2F%2Flogin.microsoftonline.com%2Fcommon%2Foauth2%2Fnativeclient">
+            Authorize Image Builder on Azure <ExternalLinkAltIcon />
+                </a>
+                <Title headingLevel="h4">Permission status</Title>
```

Comment:
With the current MVP way of authorizing our Azure application, we can't check the permission status. So maybe let's hide this for now?